Sources/Orbit/Support/Components/TimelineStep.swift

```diff
@@ -177,3 +238,94 @@ struct TimelineStepPreviews: PreviewProvider {
         .previewDisplayName("Snapshots")
     }
 }
+
+struct TimelineStepDynamicTypePreviews: PreviewProvider {
+    static var previews: some View {
+        PreviewWrapper {
+            snapshotSmall
+            snapshotLarge
+        }
+        .previewLayout(.sizeThatFits)
+    }
+
+    @ViewBuilder static var snapshotSmall: some View {
+        TimelineStepPreviews.standalone
+            .environment(\.sizeCategory, .extraSmall)
+            .previewDisplayName("Dynamic type — extraSmall")
+    }
+
+    @ViewBuilder static var snapshotLarge: some View {
+        TimelineStepPreviews.standalone
+            .environment(\.sizeCategory, .accessibilityExtraExtraLarge)
+            .previewDisplayName("Dynamic type — extra large")
+            .previewLayout(.sizeThatFits)
+    }
+}
+
+struct TimelineStepCustomContentPreviews: PreviewProvider {
+    static var previews: some View {
```

Comment:
Missing newline before the member - we probably have more of these, but we should start to clean them up to be consistent.

Reply:
Well, I will check SwiftFormatter rules. Another option will be to use Xcode templates formatted according to our guidelines.

Reply:
Our `swiftlint` has no option for this rule (regex is WIP), but another formatter might have.